release-notes/cluster-observability-operator-release-notes.adoc

```diff
@@ -26,6 +26,124 @@ The following table provides information about which features are available depe
 
 include::snippets/unified-perspective-web-console.adoc[]
 
+
+[id="cluster-observability-operator-release-notes-1-3_{context}"]
+== {coo-full} 1.3
+
+The following advisory is available for {coo-full} 1.3:
+
+//* link:https://access.redhat.com/errata/RHBA-2025:XXXX[RHBA-2025:XXXX {coo-full} 1.3]
+
+[id="cluster-observability-operator-1-3-new-features-enhancements_{context}"]
+=== New features and enhancements
+
+Incident detection feature reaches General Availability::
+The incident detection feature in the {coo-full} reaches General Availability (GA) status in this release. The incident detection feature automatically analyzes cluster health by correlating metrics, logs, and alerts to identify potential issues before they impact workloads. The health analyzer examines patterns across observability signals to detect anomalies, resource constraints, and configuration problems. With General Availability (GA) status, the incident detection feature is production-ready and fully supported for enterprise deployments. Users can enable incident detection through the monitoring `UIPlugin` resource by setting `spec.monitoring.incidents.enabled: true`.
++
+For more information, see xref:../ui_plugins/monitoring-ui-plugin.adoc#coo-incident-detection-overview_monitoring-ui-plugin[Incident detection]
++
+link:https://issues.redhat.com/browse/COO-1281[COO-1281]
+
+
+Troubleshooting panel supports silenced alerts::
```

Comment:
This is not Troubleshooting panel but Incident detection UI, part of the monitoring plugin.